element-templates/template-connector.json

```diff
@@ -41,13 +41,28 @@
         }
       },
       {
```

Comment:
Suggestion: I am wondering, whether template should also contain one dynamic variable example?

Reply:
We could add a separate template with optional authentication to showcase this.